homeassistant/components/toon/const.py

```diff
@@ -321,23 +325,23 @@
         ATTR_NAME: "Average Daily Water Usage",
         ATTR_SECTION: "water_usage",
         ATTR_MEASUREMENT: "day_average",
-        ATTR_UNIT_OF_MEASUREMENT: VOLUME_M3,
+        ATTR_UNIT_OF_MEASUREMENT: VOLUME_CUBIC_METERS,
```

Comment:
I think we should keep water out of scope for this PR and move those changes into their own PR